devices/Ssd13xx/Ssd13xx.cs

```diff
@@ -101,5 +142,172 @@ protected SpanByte SliceGenericBuffer(int start, int length)
 
             return _genericBuffer;
         }
+
+
+        /// <summary>
+        /// Draws a pixel on the screen.
+        /// </summary>
+        /// <param name="x">The x coordinate on the screen.</param>
+        /// <param name="y">The y coordinate on the screen.</param>
+        /// <param name="useWhite">Indicates if color to be used, default yes unless is inverted screen.</param>
+        public void DrawPixel(int x, int y, bool useWhite = true)
+        {
+            if ((x >= Width) || (y >= Height))
+                return;
+
+            // x specifies the column
+            int idx = x + (y / 8) * Width;
+
+            if (useWhite)
+                _genericBuffer[idx] |= (byte)(1 << (y & 7));
+            else
+                _genericBuffer[idx] &= (byte)~(1 << (y & 7));
+
+            //INVERSE COLOR: _genericBuffer[idx] ^= (byte)(1 << (y & 7));
+        }
+
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="x"></param>
+        /// <param name="y"></param>
+        /// <param name="width"></param>
+        /// <param name="height"></param>
+        /// <param name="bitmap"></param>
+        public void DrawBitmap(int x, int y, int width, int height, byte[] bitmap)
+        {
+            if ((width * height) != bitmap.Length)
+                throw new ArgumentException("Width and height do not match the bitmap size.");
+
+            for (var yO = 0; yO < height; yO++)
+            {
+                for (var xA = 0; xA < width; xA++)
+                {
+                    var b = bitmap[(yO * width) + xA];
+                    byte mask = 0x01;
```

Comment:
you may define the byte definition outside of the loop and just use mask = 0x01; here

Reply:
@gligorov you still have this one as well, declare the byte mask; before the first for so it won't get created every time